packages/cashier/src/Config/cashier-default-providers.js

```diff
@@ -0,0 +1,110 @@
+const cash_contents = [
+    { title: 'Credit/debit cards', icons: ['IcStockVisa', 'IcStockMasterCard'] },
+    {
+        title: 'Bank wire',
+        icons: [
+            'IcWalletZingpayLight',
+            'IcCashierNganLoung',
+            'IcCashierPayRetailers',
+            'IcWalletPaytrustLight',
+            'IcCashierHelpToPay',
+            'IcCashierDragonPay',
+        ],
+    },
+    {
+        title: 'E-wallets',
+        icons: [
+            'IcWalletSkrillLight',
+            'IcWalletNetellerLight',
+            'IcWalletSticpayLight',
+            'IcCashierAirTm',
+            'IcCashierPayLivre',
+            'IcWalletFasapayLight',
+            'IcWalletJetonLight',
+            'IcCashierPerfectMoney',
+        ],
+    },
+];
+const crypto_contents = [
+    {
+        title: 'Cryptos',
+        icons: ['IcCashierBitcoin', 'IcCashierEthereum', 'IcCashierLiteCoin', 'IcCashierUsdCoin', 'IcCashierTether'],
+    },
+    {
+        title: 'Buy Cryptos',
+        icons: ['IcCashierQrCode', 'IcCashierChangelly', 'IcCashierXanpoolLight', 'IcCashierBanxaLight'],
+    },
+];
+const payment_agent_contents = [
+    {
+        title: '',
+        icons: [
+            'IcCashierMandiriPay',
+            'IcCashierCimbNiaga',
+            'IcCashierBankBri',
+            'IcCashierBca',
+            'IcCashierEthereum',
+            'IcCashierBitcoin',
+            'IcCashierTether',
+        ],
+    },
+];
+const dp2p_contents = [
+    {
+        title: '',
+        icons: [
+            'IcCashierMandiriPay',
+            'IcCashierCimbNiaga',
+            'IcCashierBankBri',
+            'IcCashierBca',
+            'IcCashierEthereum',
+            'IcCashierBitcoin',
+            'IcCashierTether',
+        ],
+    },
+];
+
+const createCashProvider = onClick => {
+    return {
+        detail_click: onClick,
+        detail_description: 'Deposit cash via payment methods available in your country.',
```

Comment:
A question: do we localize these string anywhere?

Reply:
@akmal-binary Yes yes, in `cashier-default-detail.jsx` we have the translations.

Reply:
@Bahar-FS I don't think it would translate that way. If I'm not mistaken, crowdin just looks through our code for localized function/component and translates whatever is passed inside without resolving variables. Maybe @mustofa-binary can confirm if this will work or not?

Reply:
@akmal-binary Didn't know it, let me change them then.